### PR TITLE
Add data structure files from week 2 courses.

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
- Tuples
- Dictionaries
- Lists

### CODE REVIEW

**Comment by Prayoginotonegoro** [comment]
waw keren kk ..

File: ariefrahmansyah/list_2d.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+list2d = []
+list2d.append([])
+list2d.append([])
+# [ [], [] ]
+# print(list2d)
```
Comment (faisalmorensya) [inline ariefrahmansyah/list_2d.py]: remove this unused comment

**Review by faisalmorensya — CHANGES_REQUESTED** [review]

File: ariefrahmansyah/list_2d.py (hunk 1 above)
Comment (Wentyocta) [inline ariefrahmansyah/list_2d.py]: please delete unimportant comments

**Review by Wentyocta — CHANGES_REQUESTED** [review]

File: ariefrahmansyah/list_2d.py (hunk 1 above)
Comment (ariefrahmansyah) [reply ariefrahmansyah/list_2d.py]: Udah dihapus ya.

**Review by Wentyocta — APPROVED** [review]

**Pull request merged**